src/starpu/randn.cc

```diff
@@ -81,13 +81,20 @@ uint32_t footprint(struct starpu_task *task)
 }
 
 Codelet codelet_fp32, codelet_fp64, codelet_fp32_ndim0, codelet_fp64_ndim0;
+Codelet codelet_fp32_fast_tf32, codelet_fp32_fast_tf32_ndim0;
```

Comment:
`mean` and `stddev` shall be of `scal_t` type everywhere. Just like `alpha` and `beta` parameters in other functions.